lib/lotus/repository.rb

```diff
@@ -674,18 +674,18 @@ def _persisted?(entity)
 
       # Add time create an entity
       def _update_created_at(entity)
-        if entity.class.attributes.include?(:created_at)
+        if entity.respond_to?(:created_at) && entity.respond_to?(:created_at=)
           entity.created_at ||= Time.now.utc
         end
       end
 
       # Add time update an entity
       def _update_updated_at(entity)
-        if entity.class.attributes.include?(:updated_at)
-          if entity.updated_at
-            entity.updated_at = Time.now.utc
+        if entity.respond_to?(:updated_at) && entity.respond_to?(:updated_at=)
```

Comment:
@LeoTeam @mymai91 What if we add another method like `_has_timestamp?`, in order to DRY the code? :smile: 

```ruby
def _has_timestamp?(entity, name)
  entity.respond_to?(name) &&
    entity.respond_to?("#{ name }=")
end
```